sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -60,15 +62,18 @@ SELECT
   stats.impressions AS ad_impressions,
   install_dou_metrics.new_profiles_sum AS installs,
   install_dou_metrics.dau_sum AS dous,
+  install_dou_metrics.dau_sum AS dau,
```

Comment:
Do we mean to have the same value from `install_dou_metrics.dau_sum` in two separate columns by different names here?